Source: Ingmar Vanhassel
Upstream: Not yet
Reason: Don't merge over /usr/lib.

From e96e32a4bc3779f7a41076f882b25ae1a8e6c585 Mon Sep 17 00:00:00 2001
From: Ingmar Vanhassel <ingmar@exherbo.org>
Date: Wed, 28 Jan 2009 17:40:08 +0100
Subject: [PATCH] Use ${libdir} rather than ${prefix}/lib.

---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 9528cd2..12af543 100644
--- a/configure.in
+++ b/configure.in
@@ -27,7 +27,7 @@ PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.4
 AC_SUBST(GLIB_CFLAGS)
 AC_SUBST(GLIB_LIBS)
 
-CUPS_BACKEND_DIR="${prefix}/lib/cups/backend"
+CUPS_BACKEND_DIR="${libdir}/cups/backend"
 CUPS_CFLAGS="-I${includedir}/cups"
 CUPS_LIBS="-lcups"
 
-- 
1.6.1.1

